app/views/gobierto_admin/gobierto_people/people/person_posts/index.html.erb

```diff
@@ -39,7 +39,9 @@
         <%= l(person_post.created_at, format: :short) %>
       </td>
       <td>
-        <%= person_post.tags.to_sentence %>
+        <% if person_post.tags.present? %>
+          <%= person_post.tags.to_sentence %>
```

Comment:
I'm not a big fan of `#try` and it's quite possible we'll add a text such as "No tags" in a future iteration.